config/opal_check_ofi.m4

```diff
@@ -100,6 +101,17 @@ AC_DEFUN([OPAL_CHECK_OFI],[
 	AC_SUBST($1_CPPFLAGS)
 	AC_SUBST($1_LDFLAGS)
 	AC_SUBST($1_LIBS)
+    AC_ARG_ENABLE(mtl-ofi-alternative-tag,
```

Comment:
This should be in the MTL's configure.m4, not the toplevel opal_check_ofi.  This flag does nothing for OSC or a potential OOB, for example.

Reply:
Makes sense, I will move.